src/tinybox.vue

```diff
@@ -236,19 +236,22 @@ export default {
      */
     close() {
       this.goto(null);
+      this.$emit('onClose', null)
```

Comment:
We could add the index at which the image was closed:

```suggestion
      this.$emit('onClose', this.index);
      this.goto(null);
```

Reply:
The only issue with this is the event is _sent_ before tinybox actually closes itself. How about we do like this:
- copy the `this.index` to a local variable
- goto(null)
- emit close with the local variable value?